webapp/tutorials/views.py

```diff
@@ -0,0 +1,44 @@
+import flask
+import math
+
+from canonicalwebteam.discourse_docs import (
+    DiscourseAPI,
+    DiscourseDocs,
+    DocParser,
+)
+
+
+def init_tutorials(app, url_prefix):
+    discourse_parser = DocParser(
+        api=DiscourseAPI(base_url="https://forum.snapcraft.io/"),
+        index_topic_id=15409,
+        category_id=20,
```

Comment:
```suggestion
```suggestion
        category_id=34,
```
```